tests/components/firmata/test_config_flow.py

```diff
@@ -0,0 +1,51 @@
+"""Test the Firmata config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.firmata.const import CONF_SERIAL_PORT, DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.async_mock import patch
+
+
+async def test_import_cannot_connect(hass: HomeAssistant) -> None:
+    """Test we fail with an invalid board."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    with patch(
+        "homeassistant.components.firmata.board.get_board", side_effect=RuntimeError,
```

Comment:
Please patch the library and not our code. The target should be `"homeassistant.components.firmata.board.PymataExpress.start_aio"`.